README.md

```diff
@@ -68,6 +68,19 @@ app.register_task(c)
 app.run()
 ```
 
+## About dependencies
+
+This library is intended to be used to implement [Actcast application](https://actcast.io/docs/ForVendor/ApplicationDevelopment/).
+So, dependencies should be compatible with those provided by apt on the target envirenments.
```

Comment:
It's hard for me to understand what `apt` means here.
(I know, as a developer, it's apt in docker images Idein provides via actdk but actfw users may not.)

Reply:
You mean we should use the words `package manager` rather than `apt`?

Reply:
No. `apt` here sounds too general for me. But I understand this comment is for internal developers. Just slight change is acceptable.

Suggestion:

> Dependencies should be compatible with those provided by apt in Actcast application's base image to prevent installing different (even corrupt) versions.

(`So,` sounds strange here).

Reply:
I've got your point. ￼
How's 2c9be30?

Reply:
perfect 👏